src/components/Chart/Chart.jsx

```diff
@@ -161,10 +162,7 @@ const Chart = ({ digitalChannelsEnabled = false }) => {
 
     const { data, bits } = options;
 
-    let numberOfBits =
-        windowDuration <= 3000000 && showDigitalChannels
-            ? nbDigitalChannels
-            : 0;
+    let numberOfBits = showDigitalChannels ? nbDigitalChannels : 0;
```

Comment:
I assume you want to revert this change before taking this PR out of draft.

Reply:
I reverted this, but then I noticed that 3 sec limitation does not make sense in data logger. I suppose we would need a logging frequency based limitation.